### PR TITLE
CBSP-1468: Add alice and mad-hatter testing builds

### PR DESCRIPTION
**Pull request by malarky**

### CODE REVIEW

File: Top_Level_Vagrantfile [hunk 1]
```diff
@@ -165,6 +172,26 @@ couchbase_download_links = {
...
+              "ubuntu16"   => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-ubuntu16.04_amd64",
+  },
+  "mad-hatter-testing" => {
+              "centos6"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos6.x86_64",
+              "centos7"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos7.x86_64",
+              "debian8"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian8_amd64",
+              "debian9"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian9_amd64",
+              "opensuse11" => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-suse11.x86_64",
+              "opensuse12" => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-suse12.x86_64",
+              "ubuntu14"   => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-ubuntu14.04_amd64",
```
Comment (b33f) [inline Top_Level_Vagrantfile]: Ubuntu 14 won't be supported with Mad Hatter

Comment (malarky) [reply Top_Level_Vagrantfile]: Doesn't matter - we just put in whatever is being built for now.

File: Top_Level_Vagrantfile [hunk 2]
```diff
@@ -165,6 +172,26 @@ couchbase_download_links = {
...
   },
+  "alice-testing" => {
+              "centos6"    => "#{alice_builds}/couchbase-server-enterprise-6.0.0-#{alice_build_num}-centos6.x86_64",
+              "centos7"    => "#{alice_builds}/couchbase-server-enterprise-6.0.0-#{alice_build_num}-centos7.x86_64",
+              "debian8"    => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-debian8_amd64",
+              "debian9"    => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-debian9_amd64",
+              "opensuse11" => "#{alice_builds}/couchbase-server-enterprise-6.0.0-#{alice_build_num}-suse11.x86_64",
+              "opensuse12" => "#{alice_builds}/couchbase-server-enterprise-6.0.0-#{alice_build_num}-suse12.x86_64",
+              "ubuntu14"   => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-ubuntu14.04_amd64",
+              "ubuntu16"   => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-ubuntu16.04_amd64",
```
Comment (b33f) [inline Top_Level_Vagrantfile]: Need to add ubuntu18

File: Top_Level_Vagrantfile [hunk 3]
```diff
@@ -165,6 +172,26 @@ couchbase_download_links = {
...
+  },
+  "mad-hatter-testing" => {
+              "centos6"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos6.x86_64",
+              "centos7"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos7.x86_64",
+              "debian8"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian8_amd64",
+              "debian9"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian9_amd64",
+              "opensuse11" => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-suse11.x86_64",
+              "opensuse12" => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-suse12.x86_64",
+              "ubuntu14"   => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-ubuntu14.04_amd64",
+              "ubuntu16"   => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-ubuntu16.04_amd64",
```
Comment (b33f) [inline Top_Level_Vagrantfile]: Need to add ubuntu18

File: Top_Level_Vagrantfile (hunk 2 above)
Comment (malarky) [reply Top_Level_Vagrantfile]: When we start building it we can add it :-)

File: Top_Level_Vagrantfile [hunk 4]
```diff
@@ -165,6 +172,26 @@ couchbase_download_links = {
...
+              "opensuse12" => "#{alice_builds}/couchbase-server-enterprise-6.0.0-#{alice_build_num}-suse12.x86_64",
+              "ubuntu14"   => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-ubuntu14.04_amd64",
+              "ubuntu16"   => "#{alice_builds}/couchbase-server-enterprise_6.0.0-#{alice_build_num}-ubuntu16.04_amd64",
+  },
+  "mad-hatter-testing" => {
+              "centos6"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos6.x86_64",
+              "centos7"    => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-centos7.x86_64",
+              "debian8"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian8_amd64",
+              "debian9"    => "#{mad_hatter_builds}/couchbase-server-enterprise_6.5.0-#{mad_hatter_build_num}-debian9_amd64",
+              "opensuse11" => "#{mad_hatter_builds}/couchbase-server-enterprise-6.5.0-#{mad_hatter_build_num}-suse11.x86_64",
```
Comment (b33f) [inline Top_Level_Vagrantfile]: Suse11 isn't supported with Mad Hatter

**Comment by malarky** [comment]
@b33f Those changes can be made when we start building the correct packages.

**Pull request merged**